book/01-introduction/sections/about-version-control.asc

```diff
@@ -7,6 +7,7 @@ Die Dateien, die in den Beispielen in diesem Buch unter Versionsverwaltung geste
 
 Als Grafik- oder Webdesigner möchte man zum Beispiel in der Lage sein, jede Version eines Bildes oder Layouts nachverfolgen zu können. Als solcher wäre es deshalb ratsam, ein Versionsverwaltungssystem (engl. Version Control System, VCS) einzusetzen.
 Ein solches System erlaubt es, einzelne Dateien oder auch ein ganzes Projekt in einen früheren Zustand zurückzuversetzen, nachzuvollziehen, wer zuletzt welche Änderungen vorgenommen hat, die möglicherweise Probleme verursachen, herauszufinden wer eine Änderung ursprünglich vorgenommen hat und viele weitere Dinge.
+// Grunde --> Grund?
 Ein Versionsverwaltungssystem bietet allgemein die Möglichkeit, jederzeit zu einem vorherigen, funktionierenden Zustand zurückzukehren, auch wenn man einmal Mist gebaut oder aus irgendeinem Grunde Dateien verloren hat.
```

Comment:
einfach ein Typo!
```suggestion
Ein Versionsverwaltungssystem bietet allgemein die Möglichkeit, jederzeit zu einem vorherigen, funktionierenden Zustand zurückzukehren, auch wenn man einmal Mist gebaut oder aus irgendeinem Grund Dateien verloren hat.
```

Reply:
Ich fühle mich mit dieser Art zu kommentieren nicht wohl.
Mehrfach war ich im Zweifel auf welche Zeile sich dein Inline-Kommentar bezieht.

> Mein Editor löscht automatisch Leerzeichen am Ende einer Zeile ...
Dann ändere irgendwas in der betreffenden Zeile (füge z.B. ein "#" ein). Das kann beim 1. Reviewen leicht wieder entfernt werden.

Reply:
Für mich ist dein Vorschlag aufwändig, weil ich die Kommentare sowieso erst einmal in den Code schreibe, um sie mir "zu merken". Dann müsste ich vor dem Commit wieder entfernen, um sie dann wieder hier als Kommentar zu schreiben ...

Die Kommentare beziehen sich **immer** auf die nachfolgende Zeile. Ich werde mich bemühen, sie ab jetzt eindeutiger zu formulieren. Im Zweifel aber einfach hier im Kommentar nachfragen, was genau gemeint ist.